### DISCUSSION TITLE
大家的自我介绍！

### DISCUSSION THREAD

**Discussion by smd1121** [SaltyfishShop/big_discusson#5] [Information]
欢迎来到咸鱼肆！
在一切开始之前，先和大家互相认识一下吧 QWQ！

> 在鱼肆建立之初，我们的一个愿景是「联结一批愿意分享自己的笔记等作品的人，加强彼此的讨论、分享和学习」；因此我们希望大家能够通过这个 自我介绍 来为自己的作品「引流」。
> 
> 这里所说的作品可以是学习笔记的链接，**更可以是展示自己的思考以及各方面爱好的任何内容**！在这里，我们不限学校，不限身份，不限话题；我们很期待从每个人的生活或思想中得到启发！

---

### 格式要求

**【重要】** 自 2023-10-22 (Sun) 起，在本 discussion 发布、修改和删除 comment 都会被同步到 [大家的自我介绍](https://github.com/SaltyfishShop/Introduce-Yourself#readme) 仓库，并会在之后成为咸鱼肆网站中的一页。同步的方式是 PR。因此，我们对格式做要求：

- **你的自我介绍应当以 Markdown 三级标题 (`###`) 开头，并且内容中不含三级或更高级的标题，且不包含分页符 (`---`)。** 在满足这个要求的基础上，你可以随意发挥！

```markdown
### 你的名字

你的自我介绍，内容中不含三级或更高级的标题，且不包含分页符。
```

你可以在这里评论多次，但每次评论都会添加一个新的条目。因此，如果你希望介绍「另一个你」，可以新增评论；但如果想修改你已有的介绍，那么请在你原有的介绍中进行修改。你可以通过删除评论的方式将你的自我介绍从网页中删除。

---

另外，受实现所限，评论和 PR 目前都以 @smd1121 的身份发起。如果希望自己完成 PR，可以到 [大家的自我介绍](https://github.com/SaltyfishShop/Introduce-Yourself) 仓库中自己开一个。如有任何疑问或改进建议，欢迎 [提 issue](https://github.com/SaltyfishShop/Introduce-Yourself/issues)!

为防止打扰，你可以在右侧点击 Unsubscribe 来取消订阅这个 discussion 中的通知。

**Comment by smd1121** [comment]
### [咸鱼暄](https://xuan-insr.github.io/)

我叫咸鱼暄！曾经是 ZJU 19 级信息安全专业本科生，现在在乾象打工。你可以在我的 [博客](https://xuan-insr.github.io/) 找到我的联系方式！

在我的博客中有曾经写过各种专业课的笔记 / 教程，最近在写 [适合新手的 C++ 的学习笔记](https://xuan-insr.github.io/cpp/cpp_restart/)。我致力于写出让每个新人都能无痛看懂的优质教程。欢迎大家捧场！

我喜欢桌游，其他爱好还在寻找中。欢迎大家来玩！

**Reply by smd1121** [reply]
Pipeline 已创建 PR。[查看 PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/4)

**Comment by IsshikiHugh** [comment]
### Isshiki修

我是 Isshiki修！是 ZJU 21 级计算机科学与技术的本科生；目前在尝试着学习 CV/motion capture and synthesis 方向的相关内容。毕业堪忧！

是一个感觉自己应该热爱计算机技术，同时也热爱文学的思想的巨人&行动且现实的矮子！
 
以前的笔记大多更新在 [💩 yuque 数字花园](https://www.yuque.com/isshikixiu)

不过现在搭了自己的笔记本 [🔋 Isshiki修's Notebook](https://note.isshikih.top/)

搞了一个 ZJU-CS 的[资源大汇总](https://isshikihugh.github.io/zju-cs-asio/)

欢迎大家来玩！

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/7) 来将你的自我介绍合入，请耐心等待 review~

**Comment by TonyCrane** [comment]
我是 TonyCrane！是 21 图灵信安大混子。毕业堪忧！

以前的笔记更新在 [📒 鹤翔万里的笔记本](https://note.tonycrane.cc/)

现在的笔记也在 [📒 鹤翔万里的笔记本](https://note.tonycrane.cc/)（没想到吧

欢迎大家来玩！

**Reply by smd1121** [reply]
好皮的xg😍

**Reply by IsshikiHugh** [reply]
痴汉 xyx

**Comment by 27rabbit-penguin** [comment]
我是 2019 级计科学生 27rabbit，基本没有笔记 T_T （以后会有的会有的）

喜欢 数学 组合优化 体系结构

**Reply by smd1121** [reply]
你到底有几个号QAQQQ

**Comment by lhmd** [comment]
我是ZJU 21级软件工程管理和软件工程需求双料特工，目前在学习CV/三维静态重建相关内容
笔记大部分在[我的博客](http://lhmd.top/Note)
自己的一些学习资料在[一个神秘的gayhub仓库](https://github.com/lhmd/ZJU-course)
欢迎大家用JOJO踩我！

**Comment by Zhang-Each** [comment]
ZJU18级软件工程专业的学生，目前CS研一在读
发不出论文，做不完横向，找不到实习，也学不完知识
平时笔记会更新在[这个地方](https://zhang-each.github.io/My-CS-Notebook/)
其实还有个[博客](https://zhang-each.github.io/)，不过内容已经迁移到上面的笔记本中，欢迎大家多来评论区交流

**Comment by kaleo996** [comment]
大家好啊，我是<del>说的道理</del>全网不同 ID 的 ZJU 21 级 CS 人。
大概喜欢语言学吧，目前在想办法从 CS 的魔爪下抢出一些时间学自己喜欢的语言学知识，以及拓展一些 NLP、Speech 相关技术栈。
有些笔记写在咸鱼肆语雀遗址上了，以后会慢慢搬运过来的！

**Reply by kaleo996** [reply]
笔记放[个人网站](https://freshboy-cyh.github.io/)上啦！

**Comment by PinkR1ver** [comment]
看到[回顾在信安的三年](https://xuan-insr.github.io/%E7%94%9F%E6%B4%BB/%E5%9B%9E%E9%A1%BE%E5%9C%A8%E4%BF%A1%E5%AE%89%E7%9A%84%E4%B8%89%E5%B9%B4/)这个过来
大家好，我是2018级BME学生，目前BME研一在读
Personal Website：https://pinktalk.online/
着力于毕业后讨口子生活的准备

**Comment by cxzhou35** [comment]
### Zicx

ZJU 20级本科生，技能点点的很怪，对3DV和CG感兴趣

Misc:
- Homepage: https://zicx.top/home
- Notebook: https://zicx.top/notebook
- Startpage: https://zicx.top/startpage

```diff
+ 如果你喜欢 VIM，那我们就是好朋友！
+ 我用的 neovim 配置在这里：https://github.com/cxzhou35/Awesome-neovim
```

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/12) 来将你的自我介绍合入，请耐心等待 review~

**Comment by LemonApostle** [comment]
ZJU20级信息安全柠檬
有个博客，随缘更新：[note.lemons.cafe](https://note.lemons.cafe/)
兴趣在 golang 数据库 分布式

**Comment by Zerokei** [comment]
### Chrtich | 克瑞奇

ZJU20 级信息安全 Chritch
个人主页：https://zerokei.top
笔记本：https://obsidian.zerokei.top
常用软件：Obsidian / Notion
欢迎关注我的 [Github](https://github.com/Zerokei) 以及 star 我的项目！

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/29) 来将你的自我介绍合入，请耐心等待 review~

**Comment by JiepengLab** [comment]
### 𝒞ℴ𝓁𝒹_𝒾𝓃𝓀 - JiepengLab

我是𝒞ℴ𝓁𝒹_𝒾𝓃𝓀！是22级混合计科工高人！

我的**笔记本**部署在：https://note.jiepeng.tech，记录自己在计科和工高的学习~

大二上才刚开始写！希望这半年过去能完成好多高质量的笔记！
然后加好多好多友链！

搞了个可**自定义**的轻首页: https://note.jiepeng.tech/nav

欢迎大家来玩！

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/13) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/20) 来将你的自我介绍合入，请耐心等待 review~

**Comment by ruoxining** [comment]
Hi我是读本科的老奶奶芭蕉奶奶☹️
个人主页 https://ruoxining.github.io/
我的自学笔记 https://ruoxining.github.io/OBvault/
我整理的ZJU课程资源 https://github.com/ruoxining/ZJU_COURSE_MATERIALS
现在笔记不多，以后笔记也不会多的！欢迎来看！☹️

**Comment by mikamijpen** [comment]
我是dut的大三计算机学生。会点人工智能（因为常守朱），，业余喜欢看心理学等。还在卷绩点所以工程不咋会。
以下是我做的ai部分笔记。在群里名字是三上境介。
https://www.yuque.com/sanshangjingjie/search_ai
密码：dtsf

**Comment by tmplog** [comment]
ZJU21级过控学生，主要是工程、国外网课、和浙大课程资源
https://zizheng.life/
正在努力迁移中

**Comment by 27rabbitlt** [comment]
### 27rabbit
我是 2019 级计科学生 27rabbit，[27rabbitlt.github.io](https://27rabbitlt.github.io/) 有似有似无的笔记，为了学英语强迫自己写点英文内容

想要成为数学家和运维

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/5) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/22) 来将你的自我介绍合入，请耐心等待 review~

**Comment by leo-934** [comment]
### vinci
大家好，我是vinci，zju cs在读研究生。
这是我的github：[https://github.com/vinci-897](https://github.com/vinci-897)，欢迎关注。
以及，这是我的个人主页：[https://vinci-897.github.io/](https://vinci-897.github.io/)。

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/8) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/9) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/25) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/26) 来将你的自我介绍合入，请耐心等待 review~

**Comment by mid2rain** [comment]
### mid2rain
大家好，我是中二雨，zju自动化（控制）大三学生这是我的blog：[https://mid2rain.github.io/](https://mid2rain.github.io/)
本来打算做成技术向blog，结果因为各种随笔和吐槽太多，变成日常向了~~悲鸣！~~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/14) 来将你的自我介绍合入，请耐心等待 review~

**Comment by lingzhiyxp** [comment]
### 小胖一族눈_눈
大家好俺是Yuan_Xiaopang，21级自动化（电气）在读
我的notion主页：https://www.notion.so/Yuan_Lingzhi-s-cosmos-11f490b47c9a4c1c8b856d425867e558

里面放着俺追的剧&课程笔记～（有点简陋就是说）
爱好是看live听歌和干饭和追剧！

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/15) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/16) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/17) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/27) 来将你的自我介绍合入，请耐心等待 review~

**Comment by Frankgu3528** [comment]
### Frankgu

大家好，我是21级数学学院的Frankgu, 主要关注时序数据的建模。目前在做脑电波相关的任务。
这是[我的Github](https://github.com/Frankgu3528)，欢迎Follow。

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/19) 来将你的自我介绍合入，请耐心等待 review~

**Comment by HobbitQia** [comment]
### HobbitQia

大家好👋我是 ZJU 21 级图灵班 CS 专业的 HobbitQia，全网同 ID，笔记在[这里捏](https://note.hobbitqia.cc/)~  
目前对 MLsys 感兴趣（迫真  
总之欢迎来玩~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/21) 来将你的自我介绍合入，请耐心等待 review~

**Comment by CSfufu** [comment]
### fufu酱

大家好，我是fufu酱，现在是ZJU22级CS人，目前也在ITP和xlab
平时爱好划水摸鱼和摆烂，要说一点的话就是很喜欢看⚽️和🏀，巴萨&warriors
目前对fintech以及web3比较感兴趣，我的[blog](https://notion-next-six-rouge.vercel.app/)放在这里了,会放一些课程笔记技术分享，以及fintech的weekly insight
总之快来找我玩～

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/23) 来将你的自我介绍合入，请耐心等待 review~

**Comment by Dreamerryao** [comment]
### ttu

全网名多带图图(取自我的图图呢)，详细的自我介绍在 [这里](https://dreamerryao.wiki/about)

硕士在读，前端工程师 + 一点点合约 + 一点点后端 ，all in web3。

最近喜欢📷和🏸️，常驻玉泉。

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/24) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
呜呜 好喜欢你的主页

**Comment by hzeroyuke** [comment]
### 华零

大家好呀我是华零，22级软工的一只菜狗啦，自己的小博客在[这里](https://hzeroyuke.github.io/my_blog/) 
喜欢水群和桌游，喜欢和朋友一起出去玩或者干饭，同时是一只半二次元，博客未来会努力更新的（尽量克服拖延症），欢迎大家来找我玩~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/28) 来将你的自我介绍合入，请耐心等待 review~

**Comment by jzl-66666a** [comment]
### 66666a
大家好，我是66666a/GinGo，22过控er。
我的博客在[这里](https://jzl-66666a.github.io/),刚搭了个博客的架子,之后再慢慢补充吧（

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/30) 来将你的自我介绍合入，请耐心等待 review~

**Reply by smd1121** [reply]
[bot] 欢迎来到咸鱼肆！Pipeline 已创建 [PR](https://github.com/SaltyfishShop/Introduce-Yourself/pull/31) 来将你的自我介绍合入，请耐心等待 review~
